### ISSUE TITLE
Example usage

### ISSUE THREAD

**Issue by Starefossen** [Turistforeningen/node-s3-uploader#51]
Some more examples on how to use the `s3-uploader` would be nice. Maybe in an example folder, that way we can more easily integrate them with the test suite making sure no example gets broken between releases.

**Comment by anthonyringoet** [comment]
While figuring out the broken cache property in the readme was not working I was thinking about an examples folder as well yesterday!

**Comment by anthonyringoet** [comment]
Do you mean a couple of functional tests that could serve as documentation as well?
Like: is the image actually uploaded to path n on aws, are the cache headers actually set to n?

**Comment by Starefossen** [comment]
Yes. The thought was to move the [Integration Tests](https://github.com/Turistforeningen/node-s3-uploader/blob/master/test/suite.coffee#L457) as separate examples in the example directory and have a way to integrate them back into the test suite.
